README.md

```diff
@@ -11,31 +11,51 @@ Amazon Simple Email (Amazon SES) library for Delphi applications.
 
 ## Using
 
+### Using with configuration from environment
+
+A option is to get configuration from environment variables, and keep the keys out of the code.
+
+It is easy to accomplish this, simply create the three following environment variables: `AWS_REGION`, `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`. Internally, the Access Keys and Region Endpoint will be assigned from the environment variables.
+
 ```pascal
-const
-  AWS_REGION_ENDPOINT = 'email.us-west-2.amazonaws.com';
-  AWS_ACCESS_KEY = 'AKIAJQF6P3QUHRSJPZCA_EXAMPLE';
-  AWS_SECRET_KEY = 'BeVo2wwiGIg25t4jKxsqmzS3ljSxrdZfl/SJ+32K_EXAMPLE';
 var
   AmazonEmailService: TAmazonEmailService;
   Recipients: TStrings;
   FromAddress, Subject, MessageBody: string;
 begin
   Recipients := TStringList.Create;
-  Recipients.Add('email@mail.com');
-  FromAddress := 'email@mail.com';
-  Subject := 'This is the subject line with HTML.';
-  MessageBody := 'Hello. I hope you are having a good day.';
-
-  AmazonEmailService := TAmazonEmailService.Create(AWS_REGION_ENDPOINT, AWS_ACCESS_KEY, AWS_SECRET_KEY);
   try
-    AmazonEmailService.SendMail(Recipients, FromAddress, Subject, MessageBody);
+    Recipients.Add('email@mail.com');
+    FromAddress := 'email@mail.com';
+    Subject := 'This is the subject line with HTML.';
+    MessageBody := 'Hello. I hope you are having a good day.';
+
+    AmazonEmailService := TAmazonEmailService.Create;
+    try
+      AmazonEmailService.SendMail(Recipients, FromAddress, Subject, MessageBody);
+    finally
+      AmazonEmailService.Free;
+    end;
   finally
-    AmazonEmailService.Free;
+    Recipients.Free;
   end;
 end;
 ```
 
+### Using with method parameters
+
+You may also pass parameters to the constructor method:
+
+```pascal
+const
+  AWS_REGION = 'email.us-west-2.amazonaws.com';
```

Comment:
@fabioxgn is totally right, `email.us-west-2.amazonaws.com` is a endpoint not a region. See: http://docs.aws.amazon.com/general/latest/gr/rande.html